src/app/services/project_service.js

```diff
@@ -292,6 +330,32 @@ angular
         });
         return res;
     }
+    
+    function prepareModelsForSearching(models) {
+    	var transformed = [];
+    	for (var key in models) {
+    		var model = models[key];
+    		var newNode = {
+    				"node": key,
+    				"alias": model.alias,
+    				"description": model.description,
+    				"columns": [],
+    				"tags": model.tags,
+    				"raw_sql": model.raw_sql
+    			}
+    		for (var colKey in model.columns) {
+    			var column = model.columns[colKey];
+    			newNode.columns.push({
+    					"name": column.name,
+    					"description": column.description ? column.description : "",
```

Comment:
is the idea here that null/undefined values don't play nicely with fuse? Trying to understand this line and the one below it

Reply:
Thinking about it, I expect Fuse would handle nulls nicely. I just noticed that when the column information was backfilled from catalog.json instead of manifest.json, these fields came out as null instead of empty. I've been bitten enough times by things alternating between an empty string and null that I reflexively standardise them.

Reply:
fair enough! I too think that fuse will be smart enough to handle nulls/undefined values, but if it turns out that this is not the case, then happy to preserve some string-coercing code :)